Index: py3qterm-0.4/py3qterm/frontend.py
===================================================================
--- py3qterm-0.4.orig/py3qterm/frontend.py
+++ py3qterm-0.4/py3qterm/frontend.py
@@ -2,10 +2,10 @@
 import sys
 import time
 
-from qtpy.QtWidgets import QApplication, QWidget, QTabWidget, QPushButton
-from qtpy.QtCore import QTimer, QRect, Qt, Signal as pyqtSignal
-from qtpy.QtGui import (QClipboard, QPainter, QFont, QBrush, QColor,
-                            QPen, QPixmap, QImage, QContextMenuEvent)
+from PyQt5.QtWidgets import QApplication, QWidget, QTabWidget, QPushButton
+from PyQt5.QtCore import QTimer, QRect, Qt, pyqtSignal
+from PyQt5.QtGui import (QClipboard, QPainter, QFont, QBrush, QColor,
+                         QPen, QPixmap, QImage, QContextMenuEvent)
 
 from .backend import Session
 
